pocket_coffea/lib/leptons.py

```diff
@@ -42,32 +42,19 @@ def get_dilepton(electrons, muons, transverse=False):
         "charge": 0.,
     }
 
-    electrons = ak.pad_none(electrons, 2)
-    muons = ak.pad_none(muons, 2)
-
-    nelectrons = ak.num(electrons[~ak.is_none(electrons, axis=1)])
-    nmuons = ak.num(muons[~ak.is_none(muons, axis=1)])
-
-    ee = electrons[:, 0] + electrons[:, 1]
-    mumu = muons[:, 0] + muons[:, 1]
-    emu = electrons[:, 0] + muons[:, 0]
+    leptons = ak.pad_none(ak.with_name(ak.concatenate([ muons, electrons], axis=1), "PtEtaPhiMCandidate"), 2)
```

Comment:
I agree that defining leptons here make the second part of the function easier. 
For performance sake, would it make sense to concatenate maximum 2 objects per collection as it was done before? 

Can you check with a quick profiling using `--test --n-chunks 10` if padding the full collection of just 2 elements for ele and for mu does change the performance?

Reply:
I did some tests and the difference runtime is not even noticeable. Most of the events have two muons or less anyway. But I changed the code to consider only up to two muons/electrons.